docs/organizing-and-exploring-results-in-the-ui/notebooks.rst

```diff
@@ -0,0 +1,103 @@
+.. _guides-ui-notebooks:
+
+Notebooks
+=========
+
+Key Features
+------------
+
+* In Neptune, each Notebook consists of a collection of checkpoints that you upload directly from the Jupyter user interface.
+* In any project, an unlimited number of Notebooks and checkpoints is allowed.
+* You can browse checkpoints history across all |Notebooks in the project|.
+* You can |share a Notebook as a link| a Notebook as a link.
+* You can |compare two Notebooks side-by-side|, like source code.
+
+Viewing Notebook Checkpoints
+----------------------------
+
+|Youtube Video viewing|
+
+The |Notebooks tab| in the Neptune UI provides a table of all the Notebooks in the current project.
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/notebooks/nb-view-11.png
+    :target: ../_static/images/organizing-and-exploring-results-in-the-ui/notebooks/nb-view-11.png
+    :alt: image
+
+
+This view lets you see what your team members are working on, review details and checkpoints associated with a Notebook, as well as share or download a Notebook and compare two or more Notebooks.
+
+The Notebook data is arranged in the following columns:
+
+* Name
+* Owner
+* Latest checkpoint
+* Description
+
+In addition, for each Notebook, there are buttons for downloading the Notebook, comparing it with another Notebook, or for sharing a link to it.
+
+A **Compare** button at the top right displays a Notebooks Comparison pane. See :ref:`Compare Notebooks <guides-notebook-compare>`.
+
+Notebook contents
+~~~~~~~~~~~~~~~~~
+Once you select a Notebook, you can see all its contents, that is: code and markdown cells, outputs and execution count.
```

Comment:
I'm not sure if this is necessary: "that is: code and markdown cells, outputs and execution count."

People usually know how notebook looks like.